plugins/modules/cloud/rackspace/rax_clb.py

```diff
@@ -276,7 +276,7 @@ def main():
             timeout=dict(type='int', default=30),
             type=dict(choices=['PUBLIC', 'SERVICENET'], default='PUBLIC'),
             vip_id=dict(),
-            wait=dict(type='bool'),
+            wait=dict(type='bool', default=False),
```

Comment:
`wait` is passed on into `cloud_load_balancer`, and is there only tested for being truthy:
- https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/cloud/rackspace/rax_clb.py#L225
- https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/cloud/rackspace/rax_clb.py#L236
- https://github.com/ansible-collections/community.general/blob/f03ca818dae586945c5d6a0c26a031e157f18449/plugins/modules/cloud/rackspace/rax_clb.py#L255

Therefore `False` and `None` are treated the same way.